engine/src/cmd/weapon_type.cpp

```diff
@@ -0,0 +1,21 @@
+#include "weapon_type.h"
+
+WEAPON_TYPE getWeaponTypeFromString(std::string type) {
+    if(type == "Bolt")      return WEAPON_TYPE::BOLT;
+    if(type == "Ball")      return WEAPON_TYPE::BALL;
+    if(type == "Beam")      return WEAPON_TYPE::BEAM;
+    if(type == "Missile")   return WEAPON_TYPE::PROJECTILE;
+    return WEAPON_TYPE::UNKNOWN;
+}
+
+std::string getFilenameFromWeaponType(WEAPON_TYPE type) {
+    switch (type)
+    {
+    case WEAPON_TYPE::BEAM:         return "beamtexture.bmp";
+    case WEAPON_TYPE::BALL:         return "ball.ani";
+    case WEAPON_TYPE::PROJECTILE:   return "missile.bfxm";
+    case WEAPON_TYPE::UNKNOWN:
+    case WEAPON_TYPE::BOLT:
```

Comment:
please add a comment of `// fall-thru to default` (or something like that) instead of the return, just for clarity/communication so we know it's purposeful and can follow the intent.

Reply:
OK.